### DISCUSSION TITLE
[Feature] Ability to add local IP and exposed domain URLs

### DISCUSSION THREAD

**Discussion by nushankodikara** [immich-app/immich#8613] [Feature Request]
### The feature

The ability to add the local IP and exposed URL to the application so when we're on the local network, the app syncs through local IP and when we're leaving the network automatically sync through the exposed domain.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #308
